hicommonwealth/commonwealth
# Eliminate Bans table.

## Link to Issue
Closes: #8990

## Description of Changes
- Eliminates redundant "Bans" table by moving field to Address `is_banned` boolean.
- Wrote reversible migration to transfer data.
- Deleted unused getBannedAddresses route.

## Test Plan
- Make yourself admin, ban an address (note that there is currently no way to unban an address in the UI or platform). Ideally pick an address you control, but that isn't linked to your admin user.
- Switch to the banned address. Confirm error on attempting to interact with forum.
